### PR TITLE
Refresh the Tutorials!

### PR DESCRIPTION
**Pull request by cnunciato**
This replaces the Tutorials section of the Habitat web site with a newer, shinier one
with a lot more excellent content.

Signed-off-by: Christian Nunciato <cnunciato@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by cnunciato** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/260444631).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/260444631).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/260444631).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/260444631) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/260444631).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/260444631).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/260444631) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by cnunciato** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/260454436).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/260454436).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/260454436) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by reset** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by cnunciato** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/260811743).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/260811743).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/260811743) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by raskchanky** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by cnunciato** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/260831596).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/260831596).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/tutorials/sample-app/mac/build-package.html.slim [hunk 1]
```diff
@@ -49,6 +49,8 @@ section.sample-app
 
       p The package created is also stored in a relative <code>results</code> directory (for example, <code>/src/results/originname-myrailsapp-0.1.0-20170624214244-x86_64-linux.hart</code>) that persists when you exit the studio. You will mostly refer to the .hart file when uploading your package to the depot.
 
+      p <b>Note</b> If you get build errors involving nokogiri, follow their #{link_to 'troubleshooting steps', 'http://www.nokogiri.org/tutorials/installing_nokogiri.html#mac_os_x'} before re-running <code>build</code> again.
```
Comment (cnunciato) [inline www/source/tutorials/sample-app/mac/build-package.html.slim]: Shouldn't we use `blockquote Note:...` here instead to be consistent with what we have elsewhere for things like this? 

Also Nokogiri should probably be capped here.

Comment (davidwrede) [inline www/source/tutorials/sample-app/mac/build-package.html.slim]: Yup (on both accounts). New commit coming shortly.

**Comment by cnunciato** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/261207170).

**Comment by cnunciato** [comment]
Resolves #2683.

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/261207170).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by reset** [comment]
We made it!

tenor-165840994

@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/261216259).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/261216259).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**